tox_docker/tox4/log.py

```diff
@@ -0,0 +1,28 @@
+from logging import Logger
+from typing import Optional
+
+from tox.report import LOGGER, ToxHandler
+
+_HANDLER = None
+
+
+def _get_handler() -> ToxHandler:
+    global _HANDLER
+    if _HANDLER is None:
+        logger: Optional[Logger] = LOGGER
+        while logger:
+            for handler in logger.handlers:
+                if isinstance(handler, ToxHandler):
+                    _HANDLER = handler
+                    logger = None
+                    break
+            else:
+                logger = logger.parent if logger.propagate else None
+
+    assert _HANDLER is not None
```

Comment:
This here is mostly private, can you explain why would you need this?

Reply:
I couldn't figure how else to produce output in a style that's consistent with what tox itself prints.

Reply:
my mistake -- closing this thread per https://github.com/tox-dev/tox-docker/pull/113#discussion_r720482493